### ISSUE TITLE
Uikit 3 (beta 6): Problem Tab component

### ISSUE THREAD

**Issue by AlexandreZaytsev** [uikit/uikit#2332]
test with local windows 7 Fifefox - Ok
test with unix server Firefox - dont work - the system hangs, JS not working

test page
```
<!DOCTYPE html>
<html lang="en-gb">
 <head>
  <meta charset="utf-8">
  <link rel="stylesheet" href="css/uikit.min.css" />
  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
  <script src="js/uikit.min.js"></script>
 </head>
 <body>

  <nav class="uk-navbar-container uk-margin" uk-navbar>
   <div class="nav-overlay uk-navbar-right">
    <a class="uk-navbar-toggle" uk-search-icon uk-toggle="target: .nav-overlay; animation: uk-animation-fade" href="#"></a>
   </div>
   <div class="nav-overlay uk-navbar-left uk-flex-1" hidden>
    <div class="uk-navbar-item uk-width-1-1">
     <form class="uk-search uk-search-navbar uk-width-1-1">
      <input class="uk-search-input" type="search" placeholder="Search..." autofocus>
     </form>
    </div>
    <a class="uk-navbar-toggle" uk-close uk-toggle="target: .nav-overlay; animation: uk-animation-fade" href="#"></a>
   </div>
  </nav>

  <div class="uk-container">
   <div class="uk-grid-small uk-margin-small-top uk-margin-small-bottom uk-grid-match" uk-grid >
    <div class="uk-width-1-4@m">
     <div class="uk-card uk-card-default uk-card-body uk-padding-small" style="border-color: rgb(221, 221, 221); background: rgb(255, 255, 255);"> 								
     </div>			
    </div>			
    <div class="uk-width-3-4@m">
     <div class="uk-card uk-card-default uk-card-body">
      <div class="uk-grid-collapse" uk-grid>
       <div class="uk-width-3-4"><div class="uk-card">3-4</div></div>
       <div class="uk-width-1-4">
        <div class="uk-card">
	 <div>
	  <ul class="uk-tab-right" uk-tab>
           <li class="uk-active"><a href="#">Right</a></li>
	   <li><a href="#">Item</a></li>
	   <li><a href="#">Item</a></li>
	  </ul>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </div>
  </div>    
    
 </body>
</html>
```
ps
I can show example on the server

**Comment by AlexandreZaytsev** [comment]
test with http://codepen.io/anon/pen/KaZqdw - dont work
```
  <div class="uk-container">
   <div class="uk-grid-small uk-margin-small-top uk-margin-small-bottom uk-grid-match" uk-grid >
    <div class="uk-width-1-4@m">
     <div class="uk-card uk-card-default uk-card-body uk-padding-small" > 								
     </div>			
    </div>			
    <div class="uk-width-3-4@m">
     <div class="uk-card uk-card-default uk-card-body">
      <div class="uk-grid-collapse" uk-grid>
       <div class="uk-width-3-4"><div class="uk-card">3-4</div></div>
       <div class="uk-width-1-4">
        <div class="uk-card">
	 <div>
	  <ul class="uk-tab-right" uk-tab>
           <li class="uk-active"><a href="#">Right</a></li>
	   <li><a href="#">Item</a></li>
	   <li><a href="#">Item</a></li>
	  </ul>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </div>
  </div>
```

**Comment by qwertytech** [comment]
I get the same issue with Beta9 as well.

**Comment by AlexandreZaytsev** [comment]
in beta 11 the problem (.uk-tab-left or .uk-tab-right) still remains, please tell me when will be fixed?

**Comment by janschoenherr** [comment]
Fixed in 246320e28455721424a4aa910568594ab3176923
